pages/badges/badge-single.jsx

```diff
@@ -16,16 +16,19 @@ var React = require('react'),
     Modal = require('../../components/modal.jsx');
 
 var Navigation = React.createClass({
+  contextTypes: {
+    intl: React.PropTypes.object
+  },
   render: function() {
     var prev = this.props.prev;
     var next = this.props.next;
     return (
       <div className="badge-navigation">
-        <a className="previous" href={prev.url}>
+        <a className="previous" href={"/" + this.context.intl.locale + prev.url}>
           <img src={prev.img} />
           <span clasName="label">← {prev.title}</span>
         </a>
-        <a className="next" href={next.url}>
+        <a className="next" href={"/" + this.context.intl.locale + next.url}>
```

Comment:
it also feels like this is unnecessarily not-DRY: we can easily write a component `<LocaleLink>` such that we can simply pass in the urls you see in routes.jsx, without having to worry about whether you forgot the local prefix or not. If that is out of scope for this PR: that absolutely should be filed as followup so we're not introducing a new way in which to break the code with a simple typo or forgetting about localization.